konvoy: Update docs for konvoy release (#2784)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/latest/logging/index.md

    * Yes, overwrite all

<p class="message--note"><strong>NOTE: </strong> If any visualization or dashboard did not import, the issue is most likely a field that is referenced in the visualization but not indexed. Make sure that you are shipping the correct logs and that the Kibana mapping is refreshed.</p>

# Configuring elasticsearch curator addon for log maintenance

> Logs, if let to run without log maintenance, would fill up the disk space. For this purpose, Konvoy (version > 1.3.0) has elasticsearch curator as an addon. Curator is a cronjob that periodically garbage collects the indices which are older than the configured time.

Reviewer: ```suggestion
If you do not perform log maintenance, logs will fill up the disk space. For this purpose, Konvoy (version > 1.3.0) has elasticsearch curator as an addon. Curator is a cronjob that periodically garbage collects indices older than the configured time.
```

> Customer can enable curator addon with default configuration by modifying the `cluster.yaml` file:

Reviewer: ```suggestion
Customers can enable the curator addon, with default configuration, by modifying the `cluster.yaml` file:
```

```yaml
- name: elasticsearch-curator
  enabled: true
```

> The default configuration would run curator job every day at mid-night and would garbage collect all the indices which are older than 30 days. Customer can configure both the cronjob schedule and the number of days by modifying the `cluster.yaml` file:

Reviewer: ```suggestion
The default configuration runs the curator job every day at mid-night and garbage collects all indices older than 30 days. Customers can configure both the cronjob schedule and the number of days by modifying the `cluster.yaml` file:
```
